### ISSUE TITLE
Differentiate levels of semantic errors depending on the presence of not resolved copybooks

### ISSUE THREAD

**Issue by temanbrcom** [eclipse/che-che4z-lsp-for-cobol#322]
## Description ##
As a USER, I would like to have different severity levels for semantic errors if either I have missing copybooks or not, so I can understand if those errors may be solved by providing a missing copybook

## Acceptance Criteria ##
- If all the copybooks are resolved, but the semantic error (e.g. variable not defined) still present, then the severity level should be "Error" 
- If there are missing copybooks, then those errors may possibly be solved and the severity should be "Warning"
- The error messages remain the same

## Definition of Done ##
- The story is implemented according to the description.
- Unit testing done (Preferably by TDD approach, coverage should be at least 70%).
- Code review done by at least one other team member.
- Manual testing is done.
- Documentation is added and reviewed (if needed).
- The pull request is approved.
- The story is merged into the development branch.
- Tests are performed on the development branch.

**Comment by temanbrcom** [comment]
Blocked by #480

**Comment by temanbrcom** [comment]
Agreed that the severity level for semantic errors should be always ERROR. 
To be implemented in #585
